README.md

```diff
@@ -1,8 +1,14 @@
 # jdiff
 
-`jdiff` is a lightweight Python library allowing you to examine structured data. `jdiff` provides an interface to intelligently compare JSON data objects and test for the presence (or absence) of keys. You can also examine and compare corresponding key-values.
+`jdiff` is a lightweight Python library allowing you to examine structured data. `jdiff` provides an interface to intelligently compare JSON data objects in a intelligently way.
```

Comment:
Redundant statement? Why remove the previous sentence?

Reply:
Yea, dup for sure. Perhaps:

jdiff` provides an interface to intelligently compare--via key presense/absense and value comparison--JSON data objects